### ISSUE TITLE
Implement LUA binding allowing sending info text to clients

### ISSUE THREAD

**Issue by nemerle** [Segs/Segs#148]
Since we also have InfoMessageCmd now:
![cohinfomsg](https://user-images.githubusercontent.com/96597/32624112-8aa6e980-c588-11e7-94b5-aab15d9e3047.png)

**Comment by broxen** [comment]
This is awesome, as it should simplify reporting to the user. it looks like this can be accessed in-game with `/imsg <number> <string>`

What does `<number>` relate to?

**Comment by nemerle** [comment]
This is a message type as defined in InfoMessageCmd.h
```cpp
enum class InfoType : uint8_t {
    COMBAT         = 1,
    DAMAGE         = 2,
    SVR_COM        = 3,
    NPC_SAYS       = 4,
    VILLAIN_SAYS   = 5,
    REGULAR        = 6,
    PRIVATE_COM    = 7,
    TEAM_COM       = 8,
    SUPERGROUP_COM = 9,
    NEARBY_COM     = 10,
    SHOUT_COM      = 11,
    REQUEST_COM    = 12,
    FRIEND_COM     = 13,
    ADMIN_COM      = 14,
    USER_ERROR     = 15,
    DEBUG_INFO     = 16,
    EMOTE          = 17,
    CHAT_TEXT      = 18,
    PROFILE_TEXT   = 19,
    HELP_TEXT      = 20,
    STD_TEXT       = 21,
};
```

**Comment by broxen** [comment]
Is this intended to replace ChatMessage? There is some overlap there.

**Comment by broxen** [comment]
This was implemented in #611 in v6
